services/sources/clacso_service.py

```diff
@@ -69,9 +69,10 @@ def get_records(
                 if record is None: continue
 
                 try:
-                    parsed_record = self.parse_clacso_record(record)
-                    if self.verify_medium_type(parsed_record.record.medium) == True and self.verify_has_part_url(parsed_record.record.has_part) == True:
+                    parsed_record = self.parse_clacso_record(record, self.OAI_NAMESPACES)
+                    if parsed_record and parsed_record.record.medium and parsed_record.record.has_part:
                         records_array.append(parsed_record)
+                        return records_array
```

Comment:
hmm if we return the records_array here, will we exit early?

Reply:
Oh I used this line to test out the ingest process for just one record being ingested properly. I'll delete this.